tests/unit/ActiveQueryTest.php

```
@@ -0,0 +1,537 @@
+
+        $this->assertSame($query, $result);
+        $this->assertEquals(['id', 'name'], $query->select);
+        $this->assertCount(1, $query->joinWith);
+    }
+
+    #[Test]
+    #[TestDox('CreateCommand uses model connection when db is null')]
+    public function testCreateCommandUsesModelConnection(): void
+    {
+        // This test verifies the connection resolution logic
+        $query = new ActiveQuery($this->testModelClass);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$query'. (undefined)

(UnusedLocalVariable)